src/lib/blueprint.ts

```diff
@@ -1,7 +1,33 @@
 import type { Openapi } from './openapi.js'
 
+interface Parameter {
+  name: string
+  type: string
```

Comment:
We need to define this as an enum. What types are we supporting? How would this be used? I suggest we actually remove this until we identify what we need.
```suggestion
```